chia/_tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -2681,58 +2684,59 @@ async def test_split_coins(wallet_environments: WalletTestFramework) -> None:
         target_coin = next(iter(await env.xch_wallet.select_coins(uint64(250_000_000_000), action_scope)))
         assert target_coin.amount == 250_000_000_000
 
-    xch_request = SplitCoins(
-        wallet_id=uint32(1),
-        number_of_coins=uint16(100),
-        amount_per_coin=uint64(100),
-        target_coin_id=target_coin.name(),
-        fee=uint64(1_000_000_000_000),  # 1 XCH
-        push=True,
+    xch_request = SplitCMD(
+        **{
+            **wallet_environments.cmd_tx_endpoint_args(env),
+            **dict(
+                id=env.wallet_aliases["xch"],
+                number_of_coins=100,
+                amount_per_coin=CliAmount(amount=uint64(100), mojos=True),
+                target_coin_id=target_coin.name(),
+                fee=1_000_000_000_000,  # 1 XCH
```

Comment:
`TransactionEndpoint` hints the fee as `uint64`, so shouldn't it be one?  Yes, in this case we have made it so mypy just throws its hands up in despair `:]` because it can't tell what anything is or should be, but that doesn't seem like a good thing to take advantage of unnecessarily.  or maybe i'm misreading here.